core/src/test/scala/kafka/raft/KafkaMetadataLogTest.scala

```diff
@@ -206,7 +217,7 @@ final class KafkaMetadataLogTest {
     val snapshotId = new OffsetAndEpoch(numberOfRecords-4, epoch)
     val log = buildMetadataLog(tempDir, mockTime)
 
-    append(log, numberOfRecords, epoch)
+    1 to numberOfRecords foreach(_ => append(log, 1, epoch))
```

Comment:
I have noticed that in Kafka we trying to avoid this Scala syntax. How about
```scala
    (1 to numberOfRecords).foreach(_ => append(log, 1, epoch))
```